src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -89,6 +89,37 @@ public class Ado : NotificationsBase, IAdo {
         return errorCodes.Any(errorStr.Contains);
     }
 
+    private static async Async.Task<OneFuzzResultVoid> ValidatePath(string project, string path, TreeStructureGroup structureGroup, WorkItemTrackingHttpClient client) {
+        var pathParts = path.Split('\\');
+        if (pathParts[0] != project) {
```

Comment:
We should probably do a case insensitive comparison

Reply:
Yep, you're right. I was under the impression that area and iteration paths were case sensitive for some reason.